script.js

```diff
@@ -1,2 +1,3 @@
 var h1El = document.querySelector('h1');
-h1El.textContent = "Dynamic Text";
\ No newline at end of file
+h1El.textContent = "Dynamic Text";
+console.log("hello2")
```

Comment:
Hey, let's make sure we're not using console.log
```suggestion
h1El.classList.add('fun');
```